runtime/sema/errors.go

```diff
@@ -1121,8 +1121,9 @@ type ImportedProgramError struct {
 
 func (e *ImportedProgramError) Error() string {
 	return fmt.Sprintf(
-		"checking of imported program `%s` failed",
+		"checking of imported program `%s` failed: %s",
 		e.Location,
+		e.CheckerError,
```

Comment:
I'm not quite sure this is OK. Normally the errors only contain the error message of themselves, and nested errors are unrolled separately by e.g. the error pretty printer. 

Including the nested error directly in the error message removes control over how the nested errors are reported (separately, just like the other errors of this current location).

Reply:
Ah sorry! I had included this line for my own debugging, didn't meant to commit it 😬 

Completely agree -- I'll remove